lib/src/components/NxVulnerabilityIndicator/__tests__/NxVulnerabilityIndicator.test.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import NxVulnerabilityIndicator, { Props } from '../NxVulnerabilityIndicator';
+import { rtlRender, runTimers, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
+
+describe('NxVulnerabilityIndicator', function() {
+  const quickRender = rtlRender<Props>(NxVulnerabilityIndicator, {});
+
+  it('renders an element with a role of graphics-symbol', async function() {
+    const el = quickRender();
+    await runTimers();
+    const vulnerabilityIndicator = el.getByRole('graphics-symbol');
+    expect(vulnerabilityIndicator).toBeInTheDocument();
+  });
+
+  it('has a fallback role of img', async function() {
+    const el = quickRender();
+    await runTimers();
+    const vulnerabilityIndicator = el.getByRole('img', { queryFallbacks: true });
+    expect(vulnerabilityIndicator).toBeInTheDocument();
+  });
+
+  it('takes precedence of severityLevel if both props are provided', async function() {
```

Comment:
Something's wrong if this is passing; the prop is called `severityRating`

Reply:
Fixed tests for `NxVulnerabilityIndicator` here: [3be4a99](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/3be4a99165a0ea9d3c233f1b1e3dba4a97999b79)